.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 dist
 yarn-error.log
 coverage
-.env
\ No newline at end of file
+.env
+.coveralls.yml
```

Comment:
@christyjacob4 I believe this can be removed now.

Reply:
Yes let me update that

Reply:
yes let me update that